app.go

```diff
@@ -69,6 +91,26 @@ func (f UpdateKey) MarshalJSON() ([]byte, error) {
 	})
 }
 
+func (app *App) getCurSor(idCursor string) ([]byte, error) {
+	type requestBody struct {
+		Id string `json:"id,string"`
+	}
+
+	req, err := app.newRequest("GET", "records/cursor", nil, "id="+idCursor)
```

Comment:
- "requestBody" variable isn't used.
- Query should be included in body.

Reply:
body query is not work with get method for the new request